torch/distributed/_tensor/examples/comm_mode_features_example.py

```diff
@@ -45,35 +47,62 @@ def __init__(self, world_size: int, rank: int) -> None:
         self.world_size = world_size
         self.rank = rank
         self.device_type = get_device_type()
+        torch.manual_seed(0)
```

Comment:
It's probably not a good idea to hide `torch.manual_seed()` inside a `__init__` function.

Also (not related to your change) it's generally a good practice to incorporate rank into the manual seed. Otherwise sharded tensors may be actually be "replicated" numeric-wise. This can hide some issues.